app/ui/client/views/app/room.js

```diff
@@ -890,7 +890,7 @@ Meteor.startup(() => {
 
 			let room = Rooms.findOne({ _id: rid }, { fields: { t: 1 } });
 
-			if (room?.t === 'l') {
+			if (room?.t === 'l' && !FlowRouter.getParam('tab')) {
```

Comment:
https://user-images.githubusercontent.com/34130764/113038498-38eb0800-91b4-11eb-98a2-7e29b93a5ff4.mp4

Hey @renatobecker The above change should have fixed this issue which you had also mentioned earlier. Its working in my local properly. Please see the above recording.

Could you please verify this again?

Reply:
@murtaza98 I'm, actually, seeing the same you shared in the video. The problem is that when you click on `Chat History` button you're redirecting the user from `Contact Center` to a  conversation rather than keeping them in the same place and just display the `Chat History` component there. 

If there is no way to do that currently, let's remove the `Chat History` button from the `Contact Profile` panel until we fix it.